packages/liquid-html-parser/src/stage-1-cst.ts

```diff
@@ -1417,16 +1420,11 @@ function toLiquidDocAST(source: string, matchingSource: string, offset: number)
       source,
       content: 2,
       isImplicit: false,
-    },
-    descriptionContent: {
-      type: ConcreteNodeTypes.TextNode,
-      value: function (this: Node) {
-        return this.sourceString.trim();
```

Comment:
We're no longer trimming here in `ast-1`.

This means that `content.value` will contain leading + trailing whitespace, which we use to determine whether an example / description were provided inline or start on their own line.

```
@description inline means it's on the same line
@description 
otherwise it can be like this
````